### PR TITLE
DietPi-Software | Nextcloud: Solve OPcache warnings on Lighttpd

### PR DESCRIPTION
**Pull request by MichaIng**
+ Adding Nextcloud Lighttpd configuration to solve OPcache admin panel warning.

Small start with Lighttpd, while testing other stuff I just got annoyed by this warnings still popping up with Lighttpd 😆.

### CODE REVIEW

File: dietpi/dietpi-software [hunk 1]
```diff
@@ -8609,6 +8609,26 @@ _EOF_
 				fi
 
 			fi
+			
+			# Lighttpd: So far just to solve OPcache warning on admin panel
+			if (( ${aSOFTWARE_INSTALL_STATE[84]} >= 1 )); then
+			
+				local lighttpd_conf=/etc/lighttpd/lighttpd.conf
```
Comment (Fourdee) [inline dietpi/dietpi-software]: Minor nit picking 😃 
```local lighttpd_conf='/etc/lighttpd/lighttpd.conf'```

**Comment by Fourdee** [comment]
@MichaIng 

Looking good, many thanks 👍 

I'll edit PR directly for https://github.com/Fourdee/DietPi/pull/1368/files#r159380311, so i can merge now.

**Review by Fourdee — APPROVED** [review]
👍

**Comment by Fourdee** [comment]
@MichaIng 

Just a thought, does this PR require any additional uninstall code for NC, under Lighttpd?

**Comment by MichaIng** [comment]
@Fourdee 
Jep your right, configuration should be removed on uninstall, and I will add note to changelog.

**Pull request merged**